src/features/DisplayControls/DrugChart/components/DrugChartView.jsx

```diff
@@ -304,7 +315,9 @@ export default function DrugChartWrapper(props) {
           <Loading withOverlay={false} />
         </div>
       ) : transformedData && transformedData.length === 0 ? (
-        <div className="no-nursing-tasks">{NoMedicationTaskMessage}</div>
+        <div className="no-nursing-tasks">
+          {errorMessage !== "" ? errorMessage : NoMedicationTaskMessage}
```

Comment:
we can also write the above code as:
errorMessage ? errorMessage : NoMedicationTaskMessage
The above would also handle null and undefined scenarios

Reply:
changes have been made